debezium-core/pom.xml

```diff
@@ -62,20 +62,10 @@
         </dependency>
         <dependency>
```

Comment:
There still is `connect-transforms`. Shouldn't this be `provided`, too? At that occasion it version could also be managed in the parent POM, using `${version.kafka}`.

Reply:
Agreed that if we can we should use parent pom and provided as much as possible.

Not sure in this particular case since I'm not very familiar with the debezium code, was just trying to clean up some cases that potentially cause problems due to plugin.path changes.